integration-cli/docker_cli_volume_test.go

```diff
@@ -25,7 +25,7 @@ func (s *DockerSuite) TestVolumeCliCreateOptionConflict(c *check.C) {
 	c.Assert(err, check.NotNil, check.Commentf("volume create exception name already in use with another driver"))
 	c.Assert(out, checker.Contains, "A volume named test already exists")
 
-	out, _ = dockerCmd(c, "volume", "inspect", "--format='{{ .Driver }}'", "test")
+	out, _ = dockerCmd(c, "volume", "inspect", "--format={{ .Driver }}", "test")
```

Comment:
sounds good - as long as its just cleaning up and not because of something funky between corba and single-quotes.